src/add-image/index.md

```diff
@@ -0,0 +1,21 @@
+# Adding images to my game page
```

Comment:
We should shorten this section. How about something like this instead?

"+# Adding images to my game page

The images on your game page will be an influencing factor in terms of getting people to commit to play and/or download your game. Many people will simply skip over a game page with no screenshots so it is vital that you post images of your game that are relevant and flattering.

Say you built your game's graphics out of clay; a photo of the models might make a cool image. Or maybe you built a custom arcade cabinet for your game, or some mockup box art; an image of that would be great.

If your game is a work-in-progress, it's essential to post images in order to get--and keep--people interested in your game. Screenshots in particular are excellent proof of progress."